app/services/imports/logs_import_service.rb

```diff
@@ -29,6 +25,16 @@ def compose_date(xml_doc, day_str, month_str, year_str)
       end
     end
 
+    CREATION_METHODS = {
+      "Bulk Upload" => "bulk upload",
+      "Manual Entry" => "single log",
+    }.freeze
+
+    def get_creation_method(xml_doc)
+      upload_method = meta_field_value(xml_doc, "upload-method")
+      CREATION_METHODS[upload_method]
+    end
```

Comment:
`get_` is quite java like, maybe we could rename the method to just `creation_method` ?